pipes/rules/common.rules

```diff
@@ -1,13 +1,26 @@
 
+import os, os.path, sys
 # imports needed for download_file() and webfile_readlines()
 import re
 # since py3 split up urllib
 try:
     from urllib.request import urlopen
 except ImportError:
     from urllib2 import urlopen
+
+# Ensure that Snakemake rules can import viral-ngs python code
+sys.path.append(os.path.realpath(os.path.join(os.path.expanduser(config['bin_dir']))))
```

Comment:
Other option is to put code into common.rules directly, rather than importing it from util.misc; that's what I was doing originally.   This keeps the wall between Snakemake rules and CLI code, but can lead to some code duplication.

Modifying a system variable like sys.path directly feels like a bit of a hack, but I can't think of trouble down the road this would cause.